packages/content-gen/docs/MESHY_API.md

```
@@ -0,0 +1,379 @@
+All image parameters accept:
+1. **Publicly accessible URL**: `https://example.com/image.png`
+2. **Base64 Data URI**: `data:image/png;base64,<base64-encoded-data>`
+
+### Task Status Values
+- `PENDING` - In queue
+- `IN_PROGRESS` - Processing
+- `SUCCEEDED` - Complete
+- `FAILED` - Error occurred
+- `CANCELED` - Cancelled by user
+
+### Streaming (SSE)
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)